TestAssets/TestProjects/FSharpTestProjects/TestLibrary/project.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
@enricosada I assume we'll eventually want to migrate these to an fsharp repo? I don't imagine you want to be blocked by CLI process in making updates to fsc?